api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -112,7 +112,7 @@
 								 referencedTableName="order_group"
 								 referencedColumnNames="order_group_id" />
     </changeSet>
-    
```

Comment:
Do you need to touch this file in any way?

Reply:
i have fixed it,,thanks